app/services/hyrax/work_uploads_handler.rb

```diff
@@ -133,14 +138,19 @@ def append_to_work(file_set)
     ##
     # @api private
     #
+    # @note the second hash overrides values in the first hash - this along with the schema validations should disallow setting arbitrary attributes
+    #
+    #
     # @return [Hash{Symbol => Object}]
-    def file_set_args(file)
-      { depositor: file.user.user_key,
-        creator: file.user.user_key,
-        date_uploaded: file.created_at,
-        date_modified: Hyrax::TimeService.time_in_utc,
-        label: file.uploader.filename,
-        title: file.uploader.filename }
+    def file_set_args(file, file_set_params = {})
```

Comment:
Interested in feedback on whether the file_set_params should override values in the built hash or not. Right now the built hash overrides values passed in, for any values that have been passed in.

Reply:
For now I've made the decision to allow the pre-filled hash to be overridden.